services/web-app/pages/developing/index.mdx

```diff
@@ -25,44 +27,109 @@ build websites and user interfaces quickly and efficiently.
 
 ### Understand the ecosystem
 
-The following diagram shows the relationships of assets and guidance across IBM. The Carbon Design
-System is often extended or complimented by additional product and digital systems.
+Carbon is a system of systems. At the epicenter lies the `core system`, with elements, assets, code,
+design kits, and guidelines intended as a base for the widest variety of situations. These assets
+are open source and can be easily extended or adapted.
+
+Teams at IBM also create `local systems` that depend on and extend the core system for specific use
+cases.
 
 ![Carbon ecosystem](/about-carbon/ecosystem.png)
 
-## Step 2: Get Started
+## Step 2: Learn about libraries
+
+`Libraries` are the means to contribute, install, and use one or many assets in products and digital
+experiences.
+
+In the past, Libraries were scattered across several websites or lived in inaccessible team
+documents. By [centralizing libraries](https://next.carbondesignsystem.com/libraries) on the new
+Carbon platform, developers and designers no longer have to spend time looking for the right set of
+assets.
+
+<Divider>
+  <Grid>
+    <Column lg={4} md={2}>
+      **Discover code packages**
+    </Column>
+    <Column lg={7} md={5}>
+      Libraries are 1:1 with code packages. All coded assets belong to a library and have a
+      maintainer.
+    </Column>
+  </Grid>
+</Divider>
+
+<Divider>
+  <Grid>
+    <Column lg={4} md={2}>
+      **Find design kits**
+    </Column>
+    <Column lg={7} md={5}>
+      Design kits compatible with coded implementations are easily discoverable in libraries. In
+      addition, libraries are a great way to look across a portfolio of assets and read additional
+      guidance provided by library maintainers.
+    </Column>
+  </Grid>
+</Divider>
+<Divider></Divider>
+
+### Core libraries
+
+Carbon is built React first, so `Carbon React` is our most up-to-date library where components
+included in the `core system` live. Assets in this library are open source and can be easily
+extended or adapted. Core components are also offered in other frameworks, including Web Components,
+Vanilla JS, Vue, Angular, and Svelte. These frameworks are mostly community maintained.
```

Comment:
I honestly think we don't even need to mention vanilla

```suggestion
Vue, Angular, and Svelte. These frameworks are mostly community maintained.
```

Reply:
Thanks for the review! I'll check with our team on Vanilla JS.

Reply:
Confirmed that we'll take out Vanilla JS since we aren't supporting it any longer, don't need to draw attention to it